src/Genre.elm

```diff
@@ -0,0 +1,57 @@
+module Genre exposing (..)
+
+import Set exposing (..)
+import Navigation exposing (..)
+import UrlParser as Url exposing (..)
+import Multiselect exposing (..)
+
+
+{-| A (String, String) for value, Description.
+This is not a union type because making those comparable is poor.
+-}
+type alias Genre =
+    ( String, String )
+
+
+toCapital : String -> String
+toCapital str =
+    String.toUpper (String.left 1 str) ++ String.dropLeft 1 str
+
+
+{-| Initializes a multiselect with an available and selected list
+-}
+genresMultiselectModel : String -> Set Genre -> Set Genre -> Multiselect.Model
```

Comment:
I'd be inclined to either leave this in `Main.elm` or go ahead and pull in the `genreSelection` view helper and give this file a `--Views` section, but having them split feels like there isn't a clear idea of how the files are split.

Reply:
I actually wanted to move the `genreSelection` view helper into here, but couldn't figure out how to have it generate an event back to the main update (i.e., MultiselectEvent)